### PR TITLE
Apply minor suggestions from promotor review; list remaining ones in TODOs

### LINKED ISSUE #3: Apply suggestions from our promotor
Apply suggestions from https://drive.google.com/file/d/1vM3UzvNf0UwqiNwRWrlV5HCltE-kwZ1L/view

### PR DESCRIPTION
**Pull request by mbalc**
Closes #3 

@styczynski @guser21 @Ogochi I wasn't present at the review session - please help with applying the remaining fixes (also make sure to sync to avoid two people doing the same work)

### CODE REVIEW

File: src/thesis.tex [hunk 1]
```diff
@@ -186,7 +191,7 @@ \chapter{Introduction}
         through which user has full control over the allocated virtual machine. Some of the well known
         services are Amazon EC2, Google Compute Engine, Microsoft Azure IAAS.
 
-        \subsection{PAAS - Platform as a service}
+        \subsection{PAAS -- Platform as a service}
```
Comment (mbalc) [inline src/thesis.tex:194]: I think double dash is much more reasonable than triple dash - see for yourself

File: src/thesis.tex [hunk 2]
```diff
@@ -72,14 +73,16 @@
...
+% TODO the above in English
 }
 
 % TODO
 % Subject classification due to ACM
 \klasyfikacja{D. Software}
 
 % Keywords list
-\keywords{system, aws, cost, optimization, heuristic, cost management, koszt, aws, optymalizacja, heurystyka kosztowa}
+\keywords{cloud, system, AWS, cost, optimization, heuristic, cost management}
```
Comment (guser21) [inline src/thesis.tex]: cloud -> could computing
cost,optimization-> cost optimization 
AWS-> Amazon Web Services (AWS) 
delete:  system, cost, (too broad)

File: src/thesis.tex [hunk 3]
```diff
@@ -72,14 +73,16 @@
 % Doctrine of classification as it states the Socrates-Erasmus:
 \dziedzina{  
 11.3 Informatyka\\
+% TODO the above in English
```
Comment (guser21) [inline src/thesis.tex]: Informatics

Comment (guser21) [inline src/thesis.tex]: 11.3 Informatics, Computer Science

Comment (mbalc) [reply src/thesis.tex]: plx commit

Comment (mbalc) [reply src/thesis.tex]: if you're sure this is official name

File: src/thesis.tex (hunk 2 above)
Comment (mbalc) [reply src/thesis.tex]: these have to comprehensively explain the thesis' aim afaik

Comment (mbalc) [reply src/thesis.tex]: so probably we'll need many more of these there - we'll probably discuss that tomorrow

File: src/thesis.tex [hunk 4]
```diff
@@ -139,7 +139,7 @@ \chapter{Introduction}
         \subsection{What is cloud computing}
 
         Cloud computing became one of the most important paradigm shifts in the area of real world software engineering.
-        It has reshaped the whole process of how applications are developed and deployed, and reduced the amount of upfront
+        It has reshaped the whole process of how applications are developed, deployed, and reduced the amount of upfront
```
Comment (mbalc) [inline src/thesis.tex]: why this change? this doesn't fit the review

Comment (mbalc) [reply src/thesis.tex]: @guser21
